target/linux/ath79/dts/qca9563_dlink_dir-869-a1.dts

```diff
@@ -0,0 +1,161 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "D-Link DIR-869 A1";
+	compatible = "dlink,dir-869-a1", "qca,qca9563";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+
+		led_power: power {
+			label = "dir-869-a1:orange:power";
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		internet {
+			label = "dir-869-a1:white:internet";
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+		};
+
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+
+		reset {
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	gpio-export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		gpio_switch_reset {
+			gpio-export,name = "dir-869-a1:reset:switch";
+			gpio-export,output = <1>;
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+		};
+	};
```

Comment:
We don't need gpio-export anymore to declare the reset pin for the external switch, the driver has the option to specify this in the `phy0` node in the dts:
  `reset-gpios = <&gpio 11 GPIO_ACTIVE_LOW>;`
It is active low here, since the driver will write the reset output to logic `0` (i.e. "not reset"), which will result in a high level on the line (as it is currently achieved by writing `1` to an active high gpio).

Please see my PR #3027 for reference, this could be changed for DIR-859 as well, I just didn't have a device for testing at that time...